### PR TITLE
Homekit option

### PR DESCRIPTION
**Pull request by CoMPaTech**
Make homekit an option (i.e. most probably will not be available from core) for beta

### CODE REVIEW

File: custom_components/plugwise/climate.py [hunk 1]
```diff
@@ -161,13 +162,14 @@ async def async_set_hvac_mode(self, hvac_mode: str) -> None:
...
         # pw-beta: feature request - mimic HomeKit behavior
-        if hvac_mode == HVAC_MODE_OFF:
-            await self.async_set_preset_mode(PRESET_AWAY)
-        if (
-            hvac_mode in [HVAC_MODE_HEAT, HVAC_MODE_COOL]
-            and self.device["active_preset"] == PRESET_AWAY
-        ):
-            await self.async_set_preset_mode(PRESET_HOME)
+        if CONF_HOMEKIT_EMULATION:
+            if hvac_mode == HVAC_MODE_OFF:
```
Comment (bouwew) [inline custom_components/plugwise/climate.py:166]: Use `self._homekit_mode` for `hvac_mode` 2x, to show that this is pw-beta code, different from Core Plugwise code.

Comment (CoMPaTech) [reply custom_components/plugwise/climate.py]: Adjusted accordingly

**Review by bouwew — DISMISSED** [review]
Looking good!

File: README.md [hunk 2]
```diff
@@ -28,8 +28,14 @@ Our [`python-plugwise`](https://github.com/plugwise/python-plugwise) python modu
 
 # Changelog
 
-## NEW FEB 2022 [0.21.1]
-- Smile: fix issue #238
```
Comment (bouwew) [inline README.md:32]: The topic-part of this line should not be deleted from the 0.21.1 release info!

Comment (CoMPaTech) [reply README.md:32]: Wrong edit

**Review by bouwew — APPROVED** [review]

**Pull request merged**